src/control/cmd/dmg/README.md

```diff
@@ -1,286 +1,66 @@
 
-Help Options:
-  -h, --help         Show this help message
+## Communications
 
-Available commands:
-  network  Perform tasks related to locally-attached network devices (aliases: n)
-  pool     Perform tasks related to DAOS pools (aliases: p)
-  service  Perform distributed tasks related to DAOS system (aliases: sv)
-  storage  Perform tasks related to locally-attached storage (aliases: st)
+The management tool is a gRPC client and uses the
+[management API](/src/control/client) to interact concurrently with
+[`daos_server`](/src/control/cmd/daos_server/README.md) processes
+running listening gRPC servers that host the control service.
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg storage --help
-Usage:
-  dmg [OPTIONS] storage <list>
+Communications take place over the management network and addresses of
+remote storage servers to connect to can be specified on the commandline
+or in the [client config file](/utils/config/daos.yml).
 
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
+For details on how gRPC communications are secured and authenticated, see the
+[Security documentation](/src/control/security/README.md#host-authentication-with-certificates).
 
-Help Options:
-  -h, --help         Show this help message
+## Functionality
 
-Available commands:
-  list  List locally-attached SCM and NVMe storage (aliases: l)
+The functionality provided by the management tool is split into
+domains which map to individual subcommands.
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg service --help
-Usage:
-  dmg [OPTIONS] service <kill-rank>
+For exhaustive subcommand listings please see output of
+`dmg [<subcommand>] --help`.
 
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
+### Network
 
-Help Options:
-  -h, --help         Show this help message
+- scan, retrieves details of available network devices.
 
-Available commands:
-  kill-rank  Terminate server running as specific rank on a DAOS pool (aliases: kr)
+### Pool
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/dmg service kill-rank --help
-Usage:
-  dmg [OPTIONS] service kill-rank [kill-rank-OPTIONS]
+- create, creates a DAOS storage pool across targets in the system.
+- destroy, destroys a given pool identified by its UUID.
+- query, queries details of given pool identified by its UUID.
+- set and get prop, manages update and retrieval of DAOS storage pool
+properties.
 
```

Comment:
The ACL commands should be noted here as well (get-acl, overwrite-acl, update-acl, delete-acl), and the ability to list pools.